packages/@aws-cdk-testing/framework-integ/test/aws-codepipeline-actions/test/integ.pipeline-elastic-beanstalk-deploy.ts

```diff
@@ -79,7 +79,7 @@ const beanstalkEnv = new elasticbeanstalk.CfnEnvironment(stack, 'beanstlk-env',
   applicationName: beanstalkApp.applicationName!,
   environmentName: 'codepipeline-test-env',
   // see https://docs.aws.amazon.com/elasticbeanstalk/latest/platforms/platforms-supported.html#platforms-supported.nodejs
-  solutionStackName: '64bit Amazon Linux 2023 v6.0.0 running Node.js 18',
+  solutionStackName: '64bit Amazon Linux 2023 v6.0.2 running Node.js 18',
```

Comment:
This was outdated and the integ test was failing: 

```
The stack named aws-cdk-codepipeline-elastic-beanstalk-deploy failed creation, it may need to be manually deleted from the AWS console: ROLLBACK_COMPLETE: Resource handler returned message: "No Solution Stack named '64bit Amazon Linux 2023 v6.0.0 running Node.js 18' found. (Service: ElasticBeanstalk, Status Code: 400, Request ID: 753eedce-7198-4f51-96ed-f4d2f06a9259)" (RequestToken: 0fa2c53a-b816-8755-5b0c-48386ab111ce, HandlerErrorCode: InvalidRequest)
  FAILED     aws-codepipeline-actions/test/integ.pipeline-elastic-beanstalk-deploy-codepipeline-elastic-beanstalk-deploy/DefaultTest (undefined/us-east-1) 207.555s
      Integration test failed: Error: Command exited with status 1